components/raftstore-v2/src/operation/command/admin/split.rs

```diff
@@ -706,17 +706,23 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
         let region_id = self.region_id();
         if self.storage().has_dirty_data() {
             let tablet_index = self.storage().tablet_index();
-            let mailbox = store_ctx.router.mailbox(region_id).unwrap();
-            let _ = store_ctx
-                .schedulers
-                .tablet_gc
-                .schedule(tablet_gc::Task::trim(
-                    self.tablet().unwrap().clone(),
-                    self.region(),
-                    move || {
-                        let _ = mailbox.force_send(PeerMsg::TabletTrimmed { tablet_index });
-                    },
-                ));
+            if let Some(mailbox) = store_ctx.router.mailbox(region_id) {
+                let _ = store_ctx
+                    .schedulers
+                    .tablet_gc
+                    .schedule(tablet_gc::Task::trim(
+                        self.tablet().unwrap().clone(),
+                        self.region(),
+                        move || {
+                            let _ = mailbox.force_send(PeerMsg::TabletTrimmed { tablet_index });
+                        },
+                    ));
+            } else {
+                // None means the node is shutdown concurrently and thus the
+                // mailboxes in router have been cleared
+                assert!(tikv_util::thread_group::is_shutdown(!cfg!(test)));
```

Comment:
You can check it with `router.is_shutdown()`.

Reply:
Good suggestion.